### DISCUSSION TITLE
How to redirect to /admin/

### DISCUSSION THREAD

**Discussion by meerzulee** [directus/directus#10391] [Archive]
Directus works on port 8055, if I type **domain.com:8055** it redirects to **domain.com:8055/admin** . 
I want to put Directus on **domain.com/admin**. I installed nginx and configured reverse proxy.
My **.env**  PUBLIC_URL is "/admin"
My **/etc/nginx/sites-available/myproject** looks like this:
```
server {
    listen      80;
    server_name domain.com;
    charset utf-8;

    #root     /var/www/myproject;
    #index   index.html index.htm;
    location /admin {
                rewrite ^/admin/(.*) /$1 break;
                proxy_pass http://localhost:8055;
        }

    location / {
        root    /var/www/myproject/;
        try_files $uri /index.html;
    }

    error_log  /var/log/nginx/vue-app-error.log;
    access_log /var/log/nginx/vue-app-access.log;
}
                    
```
I have a Vue SPA project on /var/www/myproject/, when I go to **domain.com/admin** it works but cannot load:

![image](https://user-images.githubusercontent.com/17146298/145202164-0b4da52a-667b-4044-9b7a-4cf2e389e572.png)

How can I solve this problem?

**Comment by rijkvanzanten** [comment]
Did you setup the `PUBLIC_URL` env var?

**Reply by meerzulee** [reply]
Yes, I did. After that redirection to **domain.com/admin** works correctly, but Directus doesn't load.

**Reply by rijkvanzanten** [reply]
That's most likely caused by the proxy / rewrite configuration in nginx then.. I'm no nginx expert, so not sure

**Reply by Nisthar** [reply]
@meerzulee did you solve your issue? i think i am getting the same issue.

**Reply by meerzulee** [reply]
@Nisthar I solved issue by installing Caddy(easy to setup reverse proxy) instead of Nginx, then rewrote PUBLIC_URL of directus from / to /admin. That resolved the problem.

**Reply by fayax555** [reply]
> @Nisthar I solved issue by installing Caddy(easy to setup reverse proxy) instead of Nginx, then rewrote PUBLIC_URL of directus from / to /admin. That resolved the problem.

Can you share your Caddy config file?

**Reply by meerzulee** [reply]
it's simple, Caddyfile:
example.com {
    reverse_proxy :8055
}
